bittensor/_dataset/dataset_impl.py

```diff
@@ -169,46 +175,141 @@ def __del__(self):
     def close(self):
         self.data_queue.close()
 
-    def get_random_directories(self):
-        r""" Getting directories from a random dataset_hash
-        Where a directory could be leading to a data file or a directory file 
+    def get_folder_size(self, folder):
```

Comment:
Can we add some docustrings here? Other functions are documented, let's keep it consistent :)